### ISSUE TITLE
vgg16 parameters in fluid 

### ISSUE THREAD

**Issue by moting9** [PaddlePaddle/Paddle#7590]
I print out vgg16 parameters dims received by optimizer, dataset is cifar10. There are total "60" parameters. 
But for vgg16, there are 12 conv layers, 3 fc layers,  pool layer has no parameters, plus bias, there should only 30=2x(12+3) parameters.
why are there 60 parameters in vgg16 updated by optimizer? Thanks!

param_dim: 512
param_dim: 10
param_dim: 512
param_dim: 512
param_dim: 512
param_dim: 512
param_dim: 512, 512, 3, 3
param_dim: 512
param_dim: 512
param_dim: 512
param_dim: 512
param_dim: 256
param_dim: 512
param_dim: 512
param_dim: 512, 512, 3, 3
param_dim: 512, 512
param_dim: 64
param_dim: 128
param_dim: 64
param_dim: 64
param_dim: 128
param_dim: 512
param_dim: 64, 64, 3, 3
param_dim: 256
param_dim: 512, 512, 3, 3
param_dim: 256
param_dim: 256
param_dim: 128
param_dim: 512, 512
param_dim: 128
param_dim: 256, 256, 3, 3
param_dim: 64
param_dim: 512
param_dim: 512
param_dim: 256
param_dim: 64
param_dim: 256
param_dim: 512
param_dim: 128
param_dim: 256
param_dim: 64
param_dim: 512
param_dim: 256, 128, 3, 3
param_dim: 64, 3, 3, 3
param_dim: 512, 256, 3, 3
param_dim: 512
param_dim: 256, 256, 3, 3
param_dim: 128, 64, 3, 3
param_dim: 256
param_dim: 512
param_dim: 256
param_dim: 512, 512, 3, 3
param_dim: 128
param_dim: 512, 10
param_dim: 512
param_dim: 128, 128, 3, 3
param_dim: 512
param_dim: 512
param_dim: 512, 512, 3, 3
param_dim: 512

**Comment by JiayiFeng** [comment]
I'm not sure which API are you using to get this information. But instinctively I think it is because each parameter has a corresponding gradient.

**Comment by moting9** [comment]
Thanks!
I just add print in SGD optimizer as below.
 I have a check of vgg16  benchmark in fluid, each conv has batchnorm, maybe this is the reason. the issue can be closed. Thanks.
--- a/paddle/operators/sgd_op.cc
+++ b/paddle/operators/sgd_op.cc
@@ -35,6 +35,9 @@ class SGDOp : public framework::OperatorWithKernel {
     PADDLE_ENFORCE_EQ(framework::product(lr_dims), 1,
                       "Learning rate should have 1 element");
     auto param_dim = ctx->GetInputDim("Param");
+
+    std::cout << "param_dim: " << param_dim << std::endl;
+
